core/tauri-runtime-wry/src/lib.rs

```diff
@@ -3952,3 +3962,16 @@ fn clear_window_surface(
     let _ = buffer.present();
   }
 }
+
+#[cfg(windows)]
+fn update_webview_visibility(
+  window: &Window,
+  webviews: &[WebviewWrapper],
+) -> windows::core::Result<()> {
+  let is_visible = window.is_visible() && !window.is_minimized();
```

Comment:
I think this should be passed `is_visible` as an argument, because in the case of when calling `window.hide()` we already know that it is not visible and so we don't need to call `window.is_visible()`

Reply:
Ok, but I feel like that will put the responsibility of calculating `is_visible` to the caller, which is more error prone